CatAnalyzer/plugins/TtbarDiLeptonAnalyzer.cc

```diff
@@ -455,7 +477,7 @@ void TtbarDiLeptonAnalyzer::selectElecs(const std::vector<cat::Electron>& elecs,
     //if (el.electronID("cutBasedElectronID-PHYS14-PU20bx25-V2-standalone-medium") == 0) continue;
     if ( !el.electronID("cutBasedElectronID-Spring15-25ns-V1-standalone-medium") ) continue;
     //if (!el.passConversionVeto()) continue;
-    if (!el.isPF()) continue;
+    //if (!el.isPF()) continue;
```

Comment:
Dr.roh asked me to comment out that. but I think it's just temporary. for sync. so... I'm not sure.